west.yml

```diff
@@ -141,7 +141,7 @@ manifest:
     - name: nrfxlib
       repo-path: sdk-nrfxlib
       path: nrfxlib
-      revision: 741729d16c8ab0dd537d802d480180786951acad
+      revision: pull/1078/head
```

Comment:
SHA to use: `5725f19f53c3fbdaa0ee0c079a8fecc1a5b78ae1`

Reply:
done.